airflow/www/views.py

```diff
@@ -2407,21 +2407,39 @@ def extra_links(self):
             return response
 
         task = dag.get_task(task_id)
-        try:
-            url = task.get_extra_links(dttm, link_name)
-        except ValueError as err:
-            response = jsonify({'url': None, 'error': str(err)})
-            response.status_code = 404
-            return response
-        if url:
-            response = jsonify({'error': None, 'url': url})
-            response.status_code = 200
+        if(link_names != None and len(link_names) > 0):
+            name_url_map = {}
+            status_code = 200
+            for single_name in link_names.split(','):
```

Comment:
Can you use [`Request.args`](https://tedboy.github.io/flask/generated/generated/werkzeug.Request.args.html#werkzeug.Request.args).[`getlist`](https://tedboy.github.io/flask/generated/generated/werkzeug.MultiDict.getlist.html) here?

Reply:
`link_names` is not a name that has been assigned to anywhere.

You need to update the code on line 2394 first. Make the change in the Javascript code I requested, then use

```python
        link_name = request.args.getlist('link_name')
```

This uses the [`MultiDict.getlist()` method](https://werkzeug.palletsprojects.com/en/1.0.x/datastructures/#werkzeug.datastructures.MultiDict.getlist), giving you a list of all `link_name` entries; if there are no entries you get an empty list so you can just use `if link_names:`

Note that in Python, you **do not use parentheses in an `if` statement**, and an empty list is [considered false in a boolean test](https://docs.python.org/3/library/stdtypes.html#truth-value-testing), there is no need to use `len(...) > 0` here.

You also need to update the docstring of this function.